src/analytics.ts

```diff
@@ -1,22 +1,34 @@
-export const trackEvent =
-  typeof process !== "undefined" &&
-  process.env?.REACT_APP_GOOGLE_ANALYTICS_ID &&
-  typeof window !== "undefined" &&
-  window.gtag
-    ? (category: string, action: string, label?: string, value?: number) => {
-        try {
-          window.gtag("event", action, {
-            event_category: category,
-            event_label: label,
-            value,
-          });
-        } catch (error) {
-          console.error("error logging to ga", error);
-        }
-      }
-    : typeof process !== "undefined" && process.env?.JEST_WORKER_ID
-    ? (category: string, action: string, label?: string, value?: number) => {}
-    : (category: string, action: string, label?: string, value?: number) => {
-        // Uncomment the next line to track locally
-        // console.log("Track Event", { category, action, label, value });
-      };
+export const trackEvent = (
+  category: string,
+  action: string,
+  label?: string,
+  value?: number,
+) => {
+  try {
+    // Uncomment the next line to track locally
+    // console.log("Track Event", { category, action, label, value });
+
+    if (
+      typeof process === "undefined" ||
```

Comment:
@ad1992 how is this check that was introduce two years ago working? `process.env.*` is interpolated during build. `process` will always be undefined. I wonder how are we getting any tracking data at all since this can't be working (and it's not).

Reply:
it's good we're not using `isProdEnv` check anywhere

https://github.com/excalidraw/excalidraw/blob/25bb6738ea8619021b27e072fa7707e1b6356c7b/src/utils.ts#L695-L696

Reply:
it looks like `process.env` is defined (object with all env vars), which is surprising because I thought webpack replaces specific variables, not `process.env`. If so, then we can check for `process.env` in a try/catch.

Reply:
@ad1992 also what was the check fixing? Using excal from a <script> is still using our production bundle, so what's going on there. https://github.com/excalidraw/excalidraw/pull/2824

Reply:
Hey, I think this was needed when loading Excalidraw directly in the browser via script hence the safe check was added